### DISCUSSION TITLE
Combining building datasets to improve OSM data over the world

### DISCUSSION THREAD

**Discussion by j3r3m1** [orbisgis/geoclimate#877] [Ideas]
We would like to combine building datasets to be able to use GeoClimate even for regions we currently do not have building data. For this, we need to identify reference building datasets (provided by cities or national geographical institutes) in different parts of the world (over all continents and lattitudes).

If you know reference building datasets (that contains building footprint and optionally building height) that are openly accessible please share your knowledge in this discussion. Please share with persons that may know local data any place over the world.

**Comment by j3r3m1** [comment]
@lucianaswf do you know about Brazil and more general South America ?

**Reply by lucianaswf** [reply]
For **São Paulo** there is https://geosampa.prefeitura.sp.gov.br/PaginasPublicas/_SBC.aspx#
To visualize the buildings, zoom in a little bit, then right menu: "Camadas" > "Habitação/Edificação" > "Edificação 2D" or "Edificação 3D".
To download the shape with building heights go to left menu, seventh icon (download), then select "habitação e edificação" > "edificação"). You will have to download the 96 neighborhoods individually. I did this last week, so I can share the files with you to avoid all this work.

For **Rio de Janeiro** there is h[ttps://www.data.rio/datasets/PCRJ::edifica%C3%A7%C3%B5es-2013-1/explore?location=-22.918949%2C-43.388132%2C17.00](url). I have never use it but the platform seams easier than São Paulo. There is a download button.

For other cities I really don't know...
If you have any troubles getting the data from SP and RJ, please let me know.

**Reply by lucianaswf** [reply]
There is also Lidar Data available for São Paulo.

**Reply by j3r3m1** [reply]
> download the 96 neighborhoods individually. I did this last week, so I can share the files with you to avoid all this work.

It would be great !

> Rio de Janeiro there is h[ttps://www.data.rio/datasets/PCRJ::edifica%C3%A7%C3%B5es-2013-1/explore?location=-22.918949%2C-43.388132%2C17.00](https://github.com/orbisgis/geoclimate/discussions/url). I have never use it but the platform seams ea

I will investigate, thank you for the link

**Reply by lucianaswf** [reply]
Here you can find the 96 files (per District) and also grouped by Zones (North, South etc.)
https://drive.google.com/drive/folders/10pX_IJno7vcMpkeC5hAhxK2q1V5_x3M1?usp=sharing
If you produce something for São Paulo I am very interested!

**Reply by j3r3m1** [reply]
Perfect I have downloaded by zone thanks, you can remove the link.

**Reply by lucianaswf** [reply]
Hi! Any news on this topic?
Hope to hear from you soon!

**Reply by j3r3m1** [reply]
Yes I have downloaded the data, gather them into a single file (a huge one...) and then desagregated them into about 9 bbox. The problem with the bbox is that it does not contain the whole city. If you like I can send you the big file but it really depends on how you deal with the data. It might be easier directly downloading the data (quite convenient on the platform). Let me know.

**Reply by j3r3m1** [reply]
>  a single file (a huge one...) 

About 1 Go

**Comment by j3r3m1** [comment]
@bweeding do you know about such data in Australia ?

**Reply by bweeding** [reply]
Hi Jeremy, for the state I live in (Tasmania) there are shapefiles of building footprint available (which I use in my work) via https://maps.thelist.tas.gov.au/listmap/app/list/map.

There is also lidar data available across the country, but it is patchy (https://elevation.fsdf.org.au/). I've had to reach out to government in some cases but they have provided what they have. 

Is that the sort of information you're interested in?

Cheers
Ben

**Reply by j3r3m1** [reply]
> for the state I live in (Tasmania) there are shapefiles of building footprint available (which I use in my work) via https://maps.thelist.tas.gov.au/listmap/app/list/map.

It is exactly the kind of data I was looking for, even though no building height at least building surface.

>  There is also lidar data available across the country, but it is patchy (https://elevation.fsdf.org.au/).

 I have only downloaded data for the North of Tasmania but it seems that DEM is 1 m resolution but DSM is one second ° which is clearly not sufficient to derive the building height from DEM and DSM. Where do you find data for running UMEP ?
